### PR TITLE
Dashboard. fixed spinner and notifisation bell

### PR DESCRIPTION
**Pull request by RostyslavSt**

### CODE REVIEW

File: src/components/DashboardContent/ListUsers.js [hunk 1]
```diff
@@ -17,20 +17,19 @@ class ListUsers extends React.Component {
...
     }
 
     return (
       <section className='list-users'>
-        {listAtHome.map((item, key) => {
+        {list.map((item, key) => {
           return (
             <li key={item.id}>
-              <Avatar name={item.name} />
+              <Avatar item={item} h/>
```
Comment (Kaderimon) [inline src/components/DashboardContent/ListUsers.js:32]: what is that 'h' even means

File: src/components/Avatar/Avatar.js [hunk 2]
```diff
@@ -5,14 +5,24 @@ import './Avatar.scss';
...
-        <figurecaption>{props.name}</figurecaption>
+      <div className="person-avatar-and-name">
+        <div className="avatar-image"></div>
+        <figurecaption>{props.item.name}</figurecaption>
+      </div>
+      <div className='person-at-home'>
+        <i
+          className={
+            'fa fa-home person-at-home' + (props.item.home ? '' : '__false')}
+          aria-hidden="true">
```
Comment (Kaderimon) [inline src/components/Avatar/Avatar.js:16]: something wrong with spaces and new lines

**Review by Kaderimon — APPROVED** [review]

File: src/components/Avatar/Avatar.js [hunk 3]
```diff
@@ -5,14 +5,24 @@ import './Avatar.scss';
 const Avatar = (props) => {
   return (
     <figure className="person-view">
```
Comment (totoshk) [inline src/components/Avatar/Avatar.js:7]: Why do you use the figure and figurecaption here?

File: src/components/DashboardContent/ListDevices.scss [hunk 4]
```diff
@@ -21,4 +15,14 @@
   display: flex;
   align-items: center;
   justify-content: space-between;
-}
\ No newline at end of file
+}
+
+.list-device-spinner {
+  background-color: $text-color-active;
```
Comment (totoshk) [inline src/components/DashboardContent/ListDevices.scss:21]: You should use the variable for background, but not for text.

**Review by totoshk — APPROVED** [review]
Approve, but you need to edit styles a little. It's not critical, everything works, but the best way is to correct mistakes.

**Pull request merged**